integrationtest/java/modules/pojo-standalone-elasticsearch/src/main/java/org/hibernate/search/integrationtest/java/modules/pojo/standalone/elasticsearch/service/AuthorService.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.java.modules.pojo.standalone.elasticsearch.service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+import org.hibernate.search.engine.backend.work.execution.DocumentRefreshStrategy;
+import org.hibernate.search.engine.search.query.SearchQuery;
+import org.hibernate.search.integrationtest.java.modules.pojo.standalone.elasticsearch.entity.Author;
+import org.hibernate.search.mapper.pojo.standalone.loading.SelectionLoadingStrategy;
+import org.hibernate.search.mapper.pojo.standalone.mapping.CloseableSearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.mapping.SearchMapping;
+import org.hibernate.search.mapper.pojo.standalone.session.SearchSession;
+
+public class AuthorService implements AutoCloseable {
+
+	private final Map<Integer, Author> datastore = new HashMap<>();
+	private final CloseableSearchMapping mapping;
+
+	public AuthorService() {
+		mapping = createSearchMapping();
+	}
+
+	private CloseableSearchMapping createSearchMapping() {
+		return SearchMapping.builder()
+				.property( "hibernate.search.automatic_indexing.synchronization.strategy", "sync" )
```

Comment:
This property only works for the ORM mapper at the moment; see https://docs.jboss.org/hibernate/search/6.2/reference/en-US/html_single/#_automatic_indexing

```suggestion
```

As you noticed, you need to use `.refreshStrategy( DocumentRefreshStrategy.FORCE )` instead.